middlewares/replace_path_regex.go

```diff
@@ -0,0 +1,32 @@
+package middlewares
+
+import (
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+// ReplacePathRegex is a middleware used to replace the path of a URL request with a regular expression
+type ReplacePathRegex struct {
+	Handler     http.Handler
+	Regexp      *regexp.Regexp
+	Replacement string
+}
+
+// NewReplacePathRegexHandler returns a new instance of ReplacePathRegex
+func NewReplacePathRegexHandler(regex string, replacement string, handler http.Handler) http.Handler {
+	return &ReplacePathRegex{
+		Regexp:      regexp.MustCompile(strings.TrimSpace(regex)),
```

Comment:
Coud you replace `regexp.MustCompile(strings.TrimSpace(regex))` by `exp, err := regexp.Compile(strings.TrimSpace(regex)),` ?

Bacause if an error occurs `MustCompile` panicing and stop Træfik.